src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/AbstractRazorEditorTest.cs

```diff
@@ -63,18 +74,18 @@ public override async Task InitializeAsync()
 
             // We open the Index.razor file, and wait for the SurveyPrompt component to be classified, as that
```

Comment:
Should probably fix the comment, to explain why opening the standard Index.razor isn't enough, and we want to open our custom one, even though they both have `SurveyPrompt` components in them 😛

I'm also keen to hear the answer, as I'm guessing the bit I missed when I added this was the `count: 3` but I'm not sure.

Reply:
It's not so much that the standard content isn't "Enough", it's that we want to test the Extension and not the Blazor template. So I don't want to rely on them to only ever have exactly 2 Components in there (start and end for NavMenu and self-closing SurveyPrompt). Hypothetically we could make sure to add the `SurveyPrompt` component with our exact expectation too, but I figured that was maybe being a bit more paranoid than was needed.

Reply:
To be more specific, yes the part missing here was the count three. Without that we wait for "at least one" matching classification in a page, which may not be all of them.